www/main/viewer/Slide/QuickTools.jsx

```diff
@@ -84,6 +84,8 @@ const QuickTools = ({ isMiscSlide }) => {
     let payload;
     let actionName;
     let stateName;
+    const maxFontSize = 30;
```

Comment:
@yadvirkaur let's keep this same as the max font size in settings, change it to 20.